### DISCUSSION TITLE
what is the Xschem way to define a spice ic=voltage for capacitors?

### DISCUSSION THREAD

**Discussion by olisnr** [StefanSchippers/xschem#243] [General]
value="1n ic=0" dont seem to work, the netlist has the ic missing. and a new parameter in the capacitor symbol seems also not usfull, because it will always be there...

**Comment by olisnr** [comment]
its working, i was looking at a wrong netlist or something

**Comment by StefanSchippers** [comment]
Ah, ok. As long as you use double quotes it becomes a single **value** token, whatever you write it will be in netlist.
value="1n ic=0".
